### PR TITLE
test: add checkbox_row test

### PR DESCRIPTION
**Pull request by shan-shaji**
* Add an extension method [pumpScreen] on WidgetTester

### CODE REVIEW

**Comment by Feichtmeier** [comment]
Thank you :) Looking good. Only that pubspec.lock and the "clang off" stuff but this is also changed constantly for me locally. Looks like something we might want to put on the gitignore in the future :)

File: test/widgets/checkbox_row_test.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+void main() {
+  group('- Check box test', () {
+    testWidgets(
+      "When `enabled=null` and `value=null` SizedBox is returned",
+      (WidgetTester tester) async {
+        await tester.pumpScreen(
+          widgetBuilder: () => CheckboxRow(
+            enabled: null,
+            value: null,
+            onChanged: (vaue) {},
```
Comment (mivoligo) [inline test/widgets/checkbox_row_test.dart:17]: `vaue` looks like a typo

File: test/widgets/checkbox_row_test.dart [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    testWidgets(
+      "When `enabled=true` the check box will be enabled",
+      (WidgetTester tester) async {
+        await tester.pumpScreen(
+          widgetBuilder: () => Scaffold(
+            body: CheckboxRow(
+              enabled: true,
+              value: true,
+              onChanged: (vaue) {},
```
Comment (mivoligo) [inline test/widgets/checkbox_row_test.dart:35]: same here: `vaue`

File: test/widgets/checkbox_row_test.dart (hunk 1 above)
Comment (shan-shaji) [reply test/widgets/checkbox_row_test.dart:17]: Yeah sure, will replace it

**Pull request merged**